transmart-oauth/grails-app/init/org/transmart/oauth/Application.groovy

```diff
@@ -0,0 +1,16 @@
+package org.transmart.oauth
+
+import grails.boot.*
+import grails.boot.config.GrailsAutoConfiguration
+import grails.plugins.metadata.*
+import groovy.util.logging.Slf4j
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration
+import org.springframework.boot.autoconfigure.security.SecurityFilterAutoConfiguration
+
+@Slf4j
+@EnableAutoConfiguration(exclude = [SecurityFilterAutoConfiguration])
```

Comment:
you're right, it should be @PluginSource here. @EnableAutoConfiguration should be added only to transmart-server/Application.groovy (as a workaround to some grails-spring-security-core plugin issue), I tested this and it is not breaking after a change in transmart-oauth/Application.groovy